<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caisse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="NomArticle_lineEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="PrixArticle_lineEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AjouterArticle_pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter Article</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RetirerArticle_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Retirer Article</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReinitialiserCommande_pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>500</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Réinitialiser Commande</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ArticleTaxable_checkBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Article Taxable</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="ArticlesAjoutes_listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TotalAvantTaxes_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Avant Taxes</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalDesTaxes_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total des Taxes</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalAPayer_label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total À Payer</string>
    </property>
   </widget>
   <widget class="QLabel" name="DescriptionArticle_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Description Article</string>
    </property>
   </widget>
   <widget class="QLabel" name="PrixArticle_label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix Article</string>
    </property>
   </widget>
   <widget class="QLabel" name="SousTotal_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sous Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="ArticlesAjoutes_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Articles Ajoutes</string>
    </property>
   </widget>
   <widget class="QLabel" name="Taxes_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>396</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Taxes</string>
    </property>
   </widget>
   <widget class="QLabel" name="Total_label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>396</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error_label">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>210</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
